06.bowling_object/lib/frame.rb

```diff
@@ -0,0 +1,57 @@
+# frozen_string_literal: true
+
+require_relative 'shot'
+
+class Frame
+  def initialize(idx, first_mark, second_mark = nil, third_mark = nil)
+    @idx = idx
+    @shots = [Shot.new(first_mark), Shot.new(second_mark), Shot.new(third_mark)]
+  end
+
+  def score(frames)
+    final? ? score_without_bonus : score_without_bonus + score_bonus(frames)
+  end
+
+  private
+
+  def score_without_bonus
+    @shots.sum(&:score)
+  end
+
+  def final?
+    @idx == 9
+  end
+
+  def score_bonus(frames)
```

Comment:
```suggestion
  def bonus_score(frames)
```

とした方が「ボーナス得点」と読めていいかなーと思います